src/configuration/check_schema.ts

```diff
@@ -0,0 +1,35 @@
+import * as yup from 'yup'
```

Comment:
This is essentially doing for object-based configuration what commander does for argv-based configuration - just checks it's the right shape.